### PR TITLE
Added function to plot mutations per node.

### PR DESCRIPTION
**Pull request by savitakartik**
Added thousands separator to axis labels.
Closes issue #5

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Nice - can you paste in a couple of examples with and without the max_count please? I'm wondering if we can refactor slightly to remove the duplicate calls to plt.hist

**Comment by savitakartik** [comment]
> Nice - can you paste in a couple of examples with and without the max_count please? I'm wondering if we can refactor slightly to remove the duplicate calls to plt.hist

Ah yes, should have added examples:
image
without the optional `max_num_mutations`:
image

Yes, there are multiple calls to `plt.hist` in other functions too. I'll attempt refactoring.

**Comment by jeromekelleher** [comment]
Looks great! Matplotlib can be fickle - if duplicate calls to ``hist`` is the simplest way to do things then so be it.

LMK when you're happy to merge.

File: utils.py [hunk 1]
```diff
@@ -198,5 +205,36 @@ def plot_mutations_per_site_along_seq(
         grid.ax_marg_y.remove()
         grid.fig.set_figwidth(20)
         grid.fig.set_figheight(8)
-        grid.ax_joint.set_xlabel("Position on genome")
+        grid.ax_joint.set_xlabel("Position on genome (Mb)")
         grid.ax_joint.set_ylabel("Number of mutations")
+
+    def plot_mutations_per_node(self, max_num_muts=None, show_counts=False):
```
Comment (jeromekelleher) [inline utils.py:211]: Would something like
```python
bins = None
if max_num_muts is not None:
    bins = range(max_num_muts + 1)
counts, edges, bars = plt.hist(
     self.nodes_num_mutations,
     bins=bins,
     # rest as before
)
```

Comment (savitakartik) [reply utils.py:203]: > Would something like this work?
> 
> ```python
> bins = None
> if max_num_muts is not None:
>     bins = range(max_num_muts + 1)
> counts, edges, bars = plt.hist(
>      self.nodes_num_mutations,
>      bins=bins,
>      # rest as before
> )
> ```

This works well, thanks. I'll work on nicer bin edge values later (i.e., when `bins=None`)

Comment (savitakartik) [reply utils.py:203]: Happy to merge!

**Pull request merged**